README.md

```diff
@@ -1,2 +1,30 @@
 # derp
-A first attempt at a simple LSST DRP catalog emulator
+
+A first attempt at a simple LSST DRP catalog emulator. 
+
+A number of planned DESC project will either need, or could use, a wide area mock LSST catalog, including stars, galaxies, and artifacts, with realistic errors and uncertainties, and with the correct anticipated schema, as would be generated by the LSST Facility's Data Release Production (DRP) pipeline and released to the LSST science community. Most applications will need a mock Object table, while some will need mock ForcedDIASource light curves. DC2 will provide a 300 square degree DRP catalog that we can use to train an emulator, that can then take in a much larger "truth" catalog and generate the corresponding wide area mock LSST catalog.
+
+## Limitations
+
+What makes `derp` so simple?
+* We ignore all blended objects.
```

Comment:
Meaning... we ignore the impact of blending on object detection and properties?  Or something else?  "We ignore all blended objects" could be taken to mean we literally don't emulate anything that could be blended (which to zeroth order means dropping everything - which, now that I think about it, would be a simple and easy-to-implement emulator but not terribly satisfying).

Reply:
```suggestion
* We ignore all blended objects, focusing (for now) on one-to-one friends-of-friends matches between the truth table and the DRP Object table.
```

Reply:
Gotcha.  That suggested change looks good to me.